README.md

```diff
@@ -115,6 +115,29 @@ pantheon_deploy:
       value: "{{ lookup('env', 'ENVVAR_FROM_CI') }}"
 ```
 
+As an added option you may export secrets in a JSON file:
+
+```yaml
+pantheon_deploy:
+...
+  secrets:
+    json:
```

Comment:
If we're going to add a top-level key here, we should consider changing the existing functionality to use one too for consistency. 

```yaml
secrets:
  file:
    value: "blah"

Reply:
Updated! I also updated the examples to use the new top-level key